src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -186,6 +186,15 @@ private async Task UpdateJobRecordAsync(CancellationToken cancellationToken)
                 queryParametersList.Add(Tuple.Create(KnownQueryParameterNames.ContinuationToken, progress.ContinuationToken));
             }
 
+            if (_exportJobRecord.ExportType == ExportJobType.Patient)
+            {
+                queryParametersList.Add(Tuple.Create(KnownQueryParameterNames.Type, KnownResourceTypes.Patient));
+            }
+            else if (!string.IsNullOrEmpty(_exportJobRecord.ResourceType))
+            {
+                queryParametersList.Add(Tuple.Create(KnownQueryParameterNames.Type, _exportJobRecord.ResourceType));
```

Comment:
What happens when `_exportJobRecord.ResourceType` contains invalid resource type values? Do we need to do some validation on this input?

Reply:
This is a larger issue with search in general. I've opened a new bug to track it: #1151